#goal definition
geometry_msgs/Pose pose
---
#result definition
Circle[] circles
---
#feedback
string robot_status